### PR TITLE
Add media query

### PR DESCRIPTION
**Pull request by Wils-FOURR-speed**
This part of the portfolio  was done using these:

1. Media query
2. HTML/CSS linters
3. HTML and CSS files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -52,8 +62,11 @@ <h1 class="intro">
...
         </h1>
+        <h1 class="work">
+            My Recent Works
+        </h1>
         <div class="indicator"></div>
-        <section>
+        <section class="grid-container">
             <div class="pic-holder-container">
                 <div class="holder-container">
                     <h3>Multi-Post Stories <span>Gain + Glory</span></h3>
```
Comment (AdnanOlarmmi) [inline index.html:72]: - [x] Kindly fix the spacing between this buttons to match the Figma design, like you'll soon be doing in your next job.

- Your design
image

- Figma design
image

File: index.html [hunk 2]
```diff
@@ -52,8 +62,11 @@ <h1 class="intro">
         <h1 class="works">
             My Recent<br> <span>Works</span>
         </h1>
+        <h1 class="work">
+            My Recent Works
+        </h1>
         <div class="indicator"></div>
-        <section>
+        <section class="grid-container">
```
Comment (AdnanOlarmmi) [inline index.html:69]: - [ ] Kindly consider the following to ensure a pixel perfect design.

- Your design
image

- Kindly ensure that the screen accommodates three cards, like it is in the original design.
- Kindly increase the spaces  outside the cards.
- Kindly reduce the space between the cards, both horizontally and vertically.

File: index.html [hunk 3]
```diff
@@ -159,10 +172,13 @@ <h3>Multi-Post Stories <span>Gain + Glory</span></h3>
 
         <section class="about-me">
             <div class="about">
-                <h1 class="intro">About me</h1>
+                <h1 class="intro one">About me</h1>
```
Comment (AdnanOlarmmi) [inline index.html:175]: - [ ] Kindly ensure that the highlighted text stretches across to match the design

- Your design
image

- Figma design
image

File: index.html [hunk 4]
```diff
@@ -224,21 +243,32 @@ <h4>Skills</h3>
         </section>
```
Comment (AdnanOlarmmi) [inline index.html:244]: - [ ] Kindly add a reduce the spacing above the `button` element and increase the button beneath it to match the design.

- Your design
image

- Figma design
image

File: index.html [hunk 5]
```diff
@@ -224,21 +243,32 @@ <h4>Skills</h3>
...
-                <p>
-                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
-                </p>
+            <div class="last-part">
+                <div class="remarks">
+                    <p>
+                        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                    </p>
+                </div>
+
```
Comment (AdnanOlarmmi) [inline index.html:252]: - [ ] Kindly consider the following

- Your design
image

- Figma design
image


- Kindly ensure that the screen accommodates three cards, like it is in the original design.
- Kindly increase the spaces  outside the cards.
- Kindly reduce the space between the cards, both horizontally and vertically.
- The cards aren't the same width, I noticed that the first card is bigger that the second, I'm sure you can agree to that.

File: index.html [hunk 6]
```diff
@@ -224,21 +243,32 @@ <h4>Skills</h3>
...
+            <div class="footer-objects">
+                <img src="shape3.svg" alt="shape3" class="shape3">
+                <img src="shape4.svg" alt="shape4" class="shape4">
+                <img src="shape5.svg" alt="shape5" class="shape5">
             </div>
 
-            <form method="POST" action="https://formspree.io/f/xzbqlowk" class="footer-form">
-                <input type="text" name="full name" maxlength="30" placeholder="Full name" class="name" required>
-                <input type="email" name="email" placeholder="Email address" class="email" required>
-                <textarea name="comments" cols="30" rows="5" maxlength="500" placeholder="Hey I’d like to connect to..."></textarea>
```
Comment (AdnanOlarmmi) [inline index.html]: - [ ] Kindly fix the following to have a pixel perfect design.

- Your design
image

- Figma design
image

- The text on the left hand side is meant to be centered across the form element.
- The form input and textareas are supposed to be wider than that.
- The background images aren't properly fixed, please use the `background` css property position them properly using the css `background-position` property, and other background properties, you can make your research.
- The button `Get in touch` shouldn't be in the center.
- Add spacing between `First name` and `Last name` input tags.

File: index.html [hunk 7]
```diff
@@ -247,6 +277,16 @@ <h4>Skills</h3>
                     <li><a href="#"><img src="images/angellist.svg" alt="Angellist"></a></li>
                 </ul>
             </div>
+
```
Comment (AdnanOlarmmi) [inline index.html:280]: - [ ] Kindly add the gray border above the icons,  and add a little bit of spacing to the icon.

- Your design
image

- Figma design
image

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @Wils-FOURR-speed, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Nice design.
- App is responsive.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Wils-FOURR-speed) [reply index.html:175]: Please i've tried to do this but it seems not working

File: index.html [hunk 8]
```diff
@@ -52,8 +62,11 @@ <h1 class="intro">
         <h1 class="works">
             My Recent<br> <span>Works</span>
         </h1>
+        <h1 class="work">
+            My Recent Works
+        </h1>
         <div class="indicator"></div>
-        <section>
+        <section class="grid-container">
             <div class="pic-holder-container">
```
Comment (franclobo) [inline index.html:70]: - [x] As requested by the previous CR, please 🙏🏽 correct the margins to be similar to the Figma design. We could use justify-content: center; and align-items: center; in the class `.grid-container`, and in the class `.pic-holder-container` specify the left and right margins, e.g. margin: Xpx Ypx;

File: index.html (hunk 1 above)
Comment (franclobo) [inline index.html:72]: - [x] As requested by the last CR, please correct the distance between the features buttons to be similar to the Figma design, we could do it by using justify-content: center; in the .stories class.
- **[OPTIONAL]** Kindly use the correct background color for the buttons #EBF0EE to be similar to the Figma design.

File: index.html (hunk 3 above)
Comment (franclobo) [inline index.html:175]: - [x] As requested by the last CR please 🙏🏽 correct the width of the paragraph to be similar to the Figma design, we could use width: 610px; **APPROX**.

File: index.html [hunk 9]
```diff
@@ -224,21 +243,32 @@ <h4>Skills</h3>
         </section>
```
Comment (franclobo) [inline index.html:243]: - [x] After adjusting the paragraph you will see that the "Get my resume" overlaps the `Frameworks`, kindly fix it by adding `margin: Xpx Ypx;` in the `.other-section` class to be similar to the Figma design.
- [x] Ass required by the last CR, please correct the margins of the cards by adding the margin attribute to the `.other-section` class to be similar to the Figma design 👇🏽 

| Figma design | Your design |
|---|---|
| image | image |

File: index.html [hunk 10]
```diff
@@ -224,21 +243,32 @@ <h4>Skills</h3>
...
+                    <div class="full-name">
+                        <input type="text" name="first name" maxlength="30" placeholder="First name" class="first-name">
+                        <input type="text" name="last name" maxlength="30" placeholder="Last name" class="last-name">
+                    </div>
+                    <input type="text" name="full name" maxlength="30" placeholder="Full name" class="name" required>
+                    <input type="email" name="email" placeholder="Email address" class="email" required>
+                    <textarea name="comments" cols="30" rows="5" maxlength="500" placeholder="Hey I’d like to connect to..."></textarea>
+                    <input type="submit" value="Get in touch" class="get-button1">
+                    <p>wilsanfourrspeed@gmail.com</p>
+                </form>
```
Comment (franclobo) [inline index.html:263]: - [x] As requested by the previous CR, please 🙏🏽 correct the items in the contact form section:

> The text on the left hand side is meant to be centered across the form element.
> The form input and textareas are supposed to be wider than that.
> The background images aren't properly fixed, please use the background css property position them properly using the css background-position property, and other background properties, you can make your research.
> The button Get in touch shouldn't be in the center.
> Add spacing between First name and Last name input tags

1 Figma design | Your design |
|---|---|
| image | image |

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Wils-FOURR-speed, 

# Required Changes ♻️

istockphoto-1079018918-612x612

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Best practices of HTML & CSS ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Wils-FOURR-speed** [comment]
Please I have one more review left. I hope this one goes well please 🙏

File: style.css [hunk 11]
```diff
@@ -348,10 +356,401 @@ footer {
...
+  }
+
+  .social-media {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 1rem;
+    list-style: none;
+    margin-top: -60;
+    padding: 0;
```
Comment (Ofarouq310) [inline style.css:643]: [OPTIONAL]
- [ ] For your `margin-top`, kindly note that you must put a unit after your number.

File: style.css [hunk 12]
```diff
@@ -348,10 +356,401 @@ footer {
...
   line-height: 24px;
 }
+
+.object1b,
+.object2b {
+  display: none;
+}
+
+@media (min-width: 600px) {
+  .first-part {
```
Comment (Ofarouq310) [inline style.css:373]: [OPTIONAL]
- [ ] Kindly note that for the Desktop breakpoint in the `CSS` file., the value for the `min-width` property should be in the range of `768px` or wider.

**Review by Ofarouq310 — APPROVED** [review]
Hi @Wils-FOURR-speed, ✋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🥇 

giphy

## Highlights
✅ Followed GitHub flow.
✅ Descriptive PR title.

## Optional suggestions

- Kindly consider adding a title to your project, as it seems inappropriate and unprofessional.
  
Screenshot 2023-06-11 003730



- You are doing a great job, and you are tired I can tell, but try to give some more time to lay out the elements on the page properly, refer to the `Figma` design specs to get a better understanding of the spacing, padding & margin you need. You have got what it takes 💪

| Your design | Figma Design |
| ------------- | ------------- |
|Screenshot 2023-06-11 003643 | image | 
Screenshot 2023-06-11 003656 | image



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**